<!DOCTYPE html>
<html lang="en">
<head>
    <meta charset="UTF-8">
    <title>BioE131 Final Project</title>
    <link rel="icon" href="https://ammes.org/wp-content/uploads/2016/03/herpes-e1576332070163.jpg" type="image/x-icon">
    <style>
        body {
            font-family: Arial, sans-serif;
            text-align: center;
            max-width: 1000px;
            margin: 0 auto;
            padding: 40px 20px;
            position: relative;
            min-height: 100vh; /* Ensure the body covers the entire viewport height */
        }

        h1 {
            font-size: 2.8em;
            margin-bottom: 40px;
        }

        p {
            line-height: 1.6;
            margin: 20px auto;
            max-width: 800px;
            text-align: center;
        }

        .bottom-right-button {
            position: fixed;
            bottom: 20px;
            right: 20px;
            background-color: #007BFF;
            color: white;
            border: none;
            padding: 10px 20px;
            border-radius: 5px;
            font-size: 24px;
            text-decoration: none;
            cursor: pointer;
            display: inline-flex;
            align-items: center;
            justify-content: center;
        }

        .bottom-right-button:hover {
            background-color: #0056b3;
        }

        .arrow {
            margin-left: 8px; /* Add spacing between text and arrow */
        }
    </style>
</head>
<body>
    <h1>An Exploration of Human Herpesvirus Homology</h1>
    <h2>From Team VALK</h2>
    <h4>Vienna Huang, Anya Raju, Lucile Zhu, Katie Sie</h4>
    <p>This tool will allow you to visualize the 9 main strains of the human herpesvirus along with a few substrains and zoonotic strains.</p>

    <p>
    Note on MSA view: The MSA plugin requires uploading files containing the results of the MSA alignment. You can find these files 
    <a href="https://github.com/katie123502/katie123502.github.io/tree/main/alignments">here</a>, and you can download and upload them to the static page to visualize alignments on the MSA viewer. You can also view the MSA alignment files via the default linear genome view without downloading/uploading yourself.
    </p>

    <p>Note on Protein Visualizer Plugin: In <a href="https://github.com/katie123502/BioEC131-Final-Project-VALK/blob/main/README.md">this repo</a>
        , see the section titled Protein View Exploration ðŸ‘€ for detailed instructions on how to use this plugin to visualize proteins.</p>

    <a href="https://katie123502.github.io/jbrowse2/" class="bottom-right-button">
        Proceed to JBrowse<span class="arrow">â†’</span>
    </a>
</body>
</html>
